src/internal/operators/scanInternals.ts

```diff
@@ -16,47 +16,47 @@ export function scanInternals<V, A, S>(
   seed: S,
   hasSeed: boolean,
   emitOnNext: boolean,
-  emitBeforeComplete?: undefined | true
+  emitBeforeComplete: undefined | true,
+  source: Observable<V>,
+  subscriber: Subscriber<any>
```

Comment:
Higher order function no longer needed here. Affects `reduce` and `scan` a little.